common/test/agents/DeeplyReadAgentTest.scala

```diff
@@ -1,12 +1,39 @@
 package agents
 
+import common.{Edition}
+import contentapi.ContentApiClient
 import model.dotcomrendering.OnwardCollectionResponse
+import org.scalatest.{BeforeAndAfterAll, DoNotDiscover}
 import org.scalatest.flatspec.AnyFlatSpec
 import org.scalatest.matchers.should.Matchers
+import services.OphanApi
+import test.{
+  ConfiguredTestSuite,
+  WithMaterializer,
+  WithTestApplicationContext,
+  WithTestContentApiClient,
+  WithTestExecutionContext,
+  WithTestWsClient,
+}
 
-class DeeplyReadAgentTest extends AnyFlatSpec with Matchers {
+@DoNotDiscover class DeeplyReadAgentTest
+    extends AnyFlatSpec
+    with Matchers
+    with ConfiguredTestSuite
+    with BeforeAndAfterAll
+    with WithMaterializer
+    with WithTestWsClient
+    with WithTestApplicationContext
+    with WithTestContentApiClient {
   "DeeplyReadAgent" should "initialise with trails being an empty Seq" in {
-    val agent = new DeeplyReadAgent()
-    agent.onwardsJourneyResponse.trails shouldBe Seq.empty
+    val ophanApi = new OphanApi(wsClient)
+    val contentApiClient = testContentApiClient
+    val agent = new DeeplyReadAgent(
+      contentApiClient = contentApiClient,
+      ophanApi = ophanApi,
+    )
+    Edition.all.map(edition => {
+      agent.getTrails(edition) shouldBe Seq.empty
+    })
```

Comment:
Non-blocking suggestion: Maybe in a future PR a test could be added for the case when trails are non-empty sequence. Not sure what would be a meaningful test in this case as @jlieb10 pointed out that the agent takes a while to fetch the content.

Reply:
Happy to iterate on this test and flesh it out